src/components/ResponsiveNavbar/ResponsiveNavbar.tsx

```diff
@@ -98,10 +99,26 @@ export const ResponsiveNavbar = (props: ResponsiveNavbarProps) => {
         )}
         {children ?? (
           <ul>
-            <li>ABOUT</li>
-            <li>PROJECTS</li>
-            <li>ELEMENTS</li>
-            <li>CONTACT</li>
+            <li  onClick={() => {
```

Comment:
This approach can be problematic, coz route change might take some time, So immediately closing navigation (using onClick handler) even before routing completed will give bad user experience.

**Better solution would be to use routing events:**
```ts
const router = useRouter();
useEffect(() => {
  router.events.on('routeChangeComplete', setClose)
  return () => {
    router.events.off('routeChangeComplete', setClose)
  };
}, [router.events]);
```

But that would be framework specific, we don't need to take care, manual control is enough for now.

Reply:
Yes, that is true, I noticed it. Thanks for the correction.